streams/src/test/java/org/apache/kafka/streams/state/internals/KeyValueBytesStoreWrapper.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
+
+import java.util.List;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.utils.Bytes;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.processor.ProcessorContext;
+import org.apache.kafka.streams.processor.StateStore;
+import org.apache.kafka.streams.state.KeyValueIterator;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.StateSerdes;
+
+/* A generic wrapper around the key-value bytes stores for use in simple tests where only a basic store is required */
+public class KeyValueBytesStoreWrapper<K, V> implements KeyValueStore<K, V> {
```

Comment:
I felt this was useful enough to pull out into a separate class, but perhaps this isn't the right place for it?

Reply:
It seems fine to me, although a more descriptive name wouldn't hurt.

Reply:
nit: we usually put such test util classes in `src/test/o/a/k/test` and packaged as `o.a.k.test`.